tools/ark-plugins/README.md

```diff
@@ -13,3 +13,45 @@ Run a Docker image build from the [Dockerfile](Dockerfile) as an init container
 ```bash
 ark plugin add {yourRepo/imageName:tag}
 ```
+
+## Add a plugin
+
+- New plugins must be added under [`internal/plugins`](internal/plugins) :
+
+```             
+  ├── internal                                                                  
+    ├── plugins
+      ├── backup    # new Backup Item Action plugins 
```

Comment:
for now there's no subfolders in plugins, and for now I don't see any usecase for backup actions - do you?

Reply:
I was thinking that in the future other teams might want to add their owns plugins and having these folders may provides a way of organize them. 

Originally I had `internal/plugins/backup/function.go` and  `internal/plugins/restore/function.go`. However I did not include them in this PR because of the issue with `dep status -v`.

Reply:
sure, will you also move existing plugins? it may be in followup PR